eclair-core/src/main/scala/fr/acinq/eclair/blockchain/Monitoring.scala

```diff
@@ -16,12 +16,20 @@
 
 package fr.acinq.eclair.blockchain
 
+import io.prometheus.client.exporter.HTTPServer
+import io.prometheus.client.{Counter, Gauge}
 import kamon.Kamon
 import kamon.metric.Metric
 
 object Monitoring {
 
   object Metrics {
+    //Prometheus Metrics
+    val MempoolMinFeeratePerKwPrometheus: Gauge  = Gauge.build().name("bitcoin_mempool_min_feerate_per_kw").help( "Minimum feerate (sat/kw) for a tx to be accepted in our mempool").register()
+    val CannotRetrieveFeeratesCountPrometheus: Counter  = Counter.build().name("bitcoin_rpc_feerates_error").help( "Number of failures to retrieve on-chain feerates").register()
+    val server: HTTPServer = new HTTPServer.Builder().withPort(1234).build();
```

Comment:
That's not the right way to do this. We already have metrics defined, you shouldn't create new ones. What you need to do is to ensure that all the existing metrics are correctly sent to prometheus if you define a prometheus endpoint.

You're doing it backwards: you are currently creating an in-process prometheus endpoint. What I'd like you to do is the reverse: send metrics to a prometheus endpoint that is potentially running on a remote machine.

Ideally, a node operator would provide the IP address of a prometheus endpoint in `eclair.conf`, and we would just automatically send metrics to that endpoint with minimal code additions to the [boot process](https://github.com/ACINQ/eclair/blob/9c7ddcd8745bbea9c7113457e1a1f378435347c7/eclair-node/src/main/scala/fr/acinq/eclair/Boot.scala#L46)